portfolio/src/main/webapp/script.js

```diff
@@ -122,3 +121,24 @@ function getOrder() {
   let select = document.getElementById('commentSort');
   addParameter('commentSort', select.options[select.selectedIndex].value);
 }
+
+/**
+ * Checks whether the user is logged in or not. If the user is logged in, then
+ * display the comments section. Otherwise, the comments section will remain
+ * hidden.
+ */
+function isLoggedIn() {
+  fetch('/login').then(response => response.text()).then((toDisplay) => {
+    console.log(toDisplay);
```

Comment:
If you want to keep this, print out some context as well. 
E.g. "User login status: "